tools/installers/install_s3prl.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+MAKE=make
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+torch_17_plus=$(python3 <<EOF
+from distutils.version import LooseVersion as V
+import torch
+
+if V(torch.__version__) >= V("1.7"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+python_36_plus=$(python3 <<EOF
+from distutils.version import LooseVersion as V
+import sys
+
+if V(sys.version) >= V("3.6"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+cuda_version=$(python3 <<EOF
+import torch
+if torch.cuda.is_available():
+    version=torch.version.cuda.split(".")
+    # 10.1.aa -> 101
+    print(version[0] + version[1])
+else:
+    print("")
+EOF
+)
+echo "cuda_version=${cuda_version}"
+
+if "${torch_17_plus}" && "${python_36_plus}"; then
+
+    rm -rf s3prl
+
+    # S3PRL Commit id when making this PR: `commit a3318ca43e9cdf0b33cc934b91ed37bdbf883abb`
+    git clone https://github.com/s3prl/s3prl.git
+
+else
+    echo "[WARNING] s3prl is not prepared for pytorch<1.7.0, python<3.6 now"
```

Comment:
Is this package only be downloaded? why cannot be installed ignoring the requirements `--ignore-requires-python` option? so then you can avoid adding the export line at `tools/extra_path.sh`. I observed that `kaldi_io` is used for preprocessing specific data, such as timit or voxceleb: https://github.com/s3prl/s3prl/blob/master/preprocess/timit2ark.py or https://github.com/s3prl/s3prl/blob/master/preprocess/ark2voxceleb.py

Reply:
Just download it could work. I am not sure which one is better, only download it or install it.

Reply:
In this case, if it works, my preference is only download.
If it’s working, we can also include it in the test since it would be quick.

Reply:
Same for fairseq.
But probably only download would not work for fariseq…

Reply:
> In this case, if it works, my preference is only download.
> If it’s working, we can also include it in the test since it would be quick.

Just to confirm, do we still make s3prl optional?

Reply:
Yes, I think it (keeping s3prl to be an optional package but adding a test script) would be a good idea.
I think you can do it in a later PR.
Can you keep this as an action item?

Reply:
Sure. I'll add it in the next PR, together with WSJ recipes and results.